src/style.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --dark: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  font-family: "Roboto", serif;
```

Comment:
Roboto is `sans-serif` font
set it for the entire page, add an alternative `sans-serif` font family i.e `Arial`